deploy/config/README.md

```diff
@@ -0,0 +1,5 @@
+# Wikibase Suite Deployment Kit Configuration
+
+Will directory will contain your configuration files after the first run of the Wikibase Suite Deployment Kit. Feel free to make adjustments as you see fit. Backup this folder regularly in order to restore your configuration later.
+
+Find information on how to run the Wikibase Suite Deployment Kit in the top level ../README.md
```

Comment:
I like how tidy it is to have the docs all in the root README, but I don't like forcing consumers of WBS to have to wade through the build and test related docs on first contact with the product. I think I still see deploy as an independent product that the Build Pipeline consumes, whether it is is in its own repo yet or not. TBD.

Reply:
Disregard the part about the location of the doc. I hadn't seen the whole document and thought the diff here was it. Sorry!